src/components/IndexComponents/IndexCafeteria/index.tsx

```diff
@@ -62,7 +71,9 @@ function IndexCafeteria() {
         <button
           type="button"
           className={styles.header__title}
-          onClick={handleMoreClick}
+          onClick={() => {
+            handleMoreClick();
+          }}
```

Comment:
이렇게 하면 어떻게 될까요?

```suggestion
          onClick={handleMoreClick}
```

아래와는 어떤 차이가 있을까요?

```
          onClick={handleMoreClick()}
```